public/locales/en/translation.json

```diff
@@ -323,6 +323,14 @@
         "manageaccounts": "Manage Accounts",
         "quit": "Quit"
     },
+    "webview": {
```

Comment:
it is important that you actually run `yarn i18n` so it will create this key with the default value for all language files. :)

Reply:
Merged main and generated translations